frontend/test/metabase/scenarios/joins/joins-2.cy.spec.js

```diff
@@ -180,7 +180,7 @@ describe("scenarios > question > joined questions", () => {
 
       popover().within(() => {
         enterCustomColumnDetails({
-          formula: "[Question 5 → sum] / [Sum of Rating]",
+          formula: "[Question 5 → Sum of Rating] / [Sum of Rating]",
```

Comment:
This patch fixes this bug where we were failing to find the joined field's display name in the expression editor. Is this an issue?